sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -3635,7 +3640,9 @@ piEnqueueKernelLaunch(pi_queue Queue, pi_kernel Kernel, pi_uint32 WorkDim,
 pi_result piEventCreate(pi_context Context, pi_event *RetEvent) {
   size_t Index = 0;
   ze_event_pool_handle_t ZeEventPool = {};
-  ZE_CALL(Context->getFreeSlotInExistingOrNewPool(ZeEventPool, Index));
+  if (auto Res = Context->getFreeSlotInExistingOrNewPool(ZeEventPool, Index))
```

Comment:
getFreeSlotInExistingOrNewPool() returns ze_result_t, while the result of piEventCreate is pi_result. It looks like that 'mapError' map should be applied to Res, right?

Reply:
No. I've changed the 2 methods to return pi_result instead.